src/main/java/frc/robot/Robot.java

```diff
@@ -33,6 +37,13 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     createRobotContainer();
+ 
+    if (SubsystemChecker.canSubsystemConstruct(SubsystemType.ArmSubsystem)) {
+      System.out.println("*****robotInit*****");
+      new WaitCommand(5).andThen(new SyncArmEncoders()).schedule();
+      //new WaitCommand(10).andThen(new CheckArmEncodersSync()).schedule();
```

Comment:
Both commands require the ArmSubsystem and one of them is cancelling the other.

This will detach the requirement from the WaitCommand(10):
`new WaitCommand(10).andThen(new ScheduleCommand(new CheckArmEncodersSync())).schedule();`

Reply:
added that line, it works